### ISSUE TITLE
Low-weight fonts in Ghost Admin are unreadable

### ISSUE THREAD

**Issue by vickychijwani** [TryGhost/Ghost#8136]
### Issue Summary

TL;DR: http://contrastrebellion.com/ (a bit cheeky, but gets the point across)

Hi! I have a problem with the Ghost Admin UI: most of the thin-weight text on the login and setup screens is unreadable on my aging display. I'm not exaggerating - I literally cannot read much of it. Like in the screenshot below, I can read "Welcome to Ghost", but not the caption or the button text unless I get _really_ close to the screen and squint:

![unreadable fonts](https://cloud.githubusercontent.com/assets/439616/23838739/ce63af92-07be-11e7-98fd-c60cdaffe5a1.png)

The worst offender is on the last setup screen ("Invite your team"). I once spent several minutes looking for a way to skip this step, and while I did eventually find it, I honestly cannot _see_ that text at all. Maybe it's just my monitor, but I think the blame lies with the 12px, 100-weight, #7d878a-on-white text.

![the worst offender](https://cloud.githubusercontent.com/assets/439616/23838842/aecfb030-07bf-11e7-9cb1-91ff3e84e749.png)

Here's some quick math to support my claim: taking _just the color_ into account, [the contrast ratio works out to 3.7:1](http://leaverou.github.io/contrast-ratio/#%237d878a-on-white), which already falls well short of [the W3C's recommended minimum of 4.5:1](https://www.w3.org/TR/WCAG/#visual-audio-contrast) for text smaller than 24px. Add in the super low weight and small size, and the contrast goes down even further.

Additionally, I haven't faced this problem on any other website, barring [1 minor case](https://twitter.com/vickychijwani/status/740535914657521665).

This issue may have gone unnoticed because the team probably uses MacBooks for work? 😄 The other reason could be that [Fira Sans is one of the few (or only?) listed fonts that actually _supports_ ultra low weights](https://www.fontsquirrel.com/fonts/fira-sans). Please consider increasing the contrast (e.g., a font weight of 400) or removing Fira Sans altogether. Thanks for reading!

### Technical details:

* Ghost Version: 1.0.0 alphas and 0.11.x as well
* Browser/OS: Firefox 51 on Linux
* Display: [Samsung SyncMaster B2030](http://tech.firstpost.com/reviews/samsung-syncmaster-b2030-monitor-13967.html) (2010 model) @ 1600x900 (92 dpi)
* Font family used: [Fira Sans](https://www.fontsquirrel.com/fonts/fira-sans)
* My eyesight: mostly fine - I have mild red-green color-blindness

**Comment by JohnONolan** [comment]
Wow wtf that is horrible - investigating!
